packages/msprime/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: msprime
+  version: 1.1.1
+source:
+  url: https://files.pythonhosted.org/packages/30/ac/22b9d2dc6e3233cc35e613b714d52cf87e4af00064a53d226824401629cb/msprime-1.1.1.tar.gz
+  sha256: e840bed2ba37ae572b234f0b55af006dc408881593f184304481ee6058ea6f30
+build:
+  script: |
+    pip install -t $HOSTSITEPACKAGES numpy
```

Comment:
So numpy is a build dependency as far as I understand? 

It should already be installed on when  building this package. So either it should build without this line altogether, or if it doesn't work you might need to adjust cflags/ldfags to include the right folders see for example https://github.com/pyodide/pyodide/blob/dc29ae65416e624e1fc2bc3b48c65f71a08ce99a/packages/scipy/meta.yaml#L42

Reply:
Due to the way msprime is currently built it's `setup.py` imports `numpy`. This is planned to be fixed soon, at which point I will remove this. I've opened https://github.com/tskit-dev/msprime/issues/2055 to track.

Reply:
Right, so numpy is a build time dependency -- we have many such packages packaged (e.g. scipy, pandas), it's not an issue as far as Pyodide is concerned. My previous suggestions should still apply.

Also for the future if you could avoid rebasing and add commits instead -- it's easier to see what changed since last review, and we squash commits when merging anyway. Thanks!

Reply:
This shouldn't be necessary in any case because it is already done by building the `numpy` package. So having it in the `run` requirements below covers it. Hopefully we will add `build` requirements and then `numpy` could be moved there.